jaseci_ai_kit/jaseci_ai_kit/modules/ph/utils/optimize.py

```diff
@@ -0,0 +1 @@
+# TBA
```

Comment:
Do we need this file for training to work?

Reply:
No I kept this file to add JIT optimization scripts (which are not production ready)